src/libbson/NEWS

```diff
@@ -1,3 +1,8 @@
+libbson 1.24.0 (Unreleased)
+===========================
+
+  * Support will be dropped for macOS 10.14 and 10.15 in an upcoming release.
```

Comment:
```suggestion
  * Support for macOS 10.14 will be dropped in an upcoming release.
```
The support page doesn't include/mention macOS 10.15, so its inclusion in this statement seems redundant.